src/main/scala/com.gu.gibbons/UserDidNotAnswer.scala

```diff
@@ -29,9 +29,9 @@ class UserDidNotAnswer[F[_]: Monad](
 
   def getUsers(now: OffsetDateTime): F[Vector[User]] =
     for {
-      _ <- logger.info(s"Getting all the users which have not extended their account since ${Settings.gracePeriod}")
       users <- bonobo.getInactiveUsers(now.minus(Settings.gracePeriod).toInstant)
       filteredUsers <- bonobo.getDevelopers(users)
```

Comment:
filterUsers could be called developerUsers so people would know what has been filtered into it without having to read back.

Reply:
Thanks for pointing that out Tony! I renamed it.